modules/js/cam.js

```diff
@@ -8,12 +8,15 @@ let pivotValue = 0;
 let c = 432; //Connector Length for open-close
 let originalWidth1;
 let originalWidth2;
+const windowWidthConst = 0.75 * 0.5;
+const windowHeightConst = 0.75;
```

Comment:
I would probably use one of 2 conventions for constants.  k prefix or screaming snake.
So instead of windowWidthConst, I would use either:
kWindowWidth = 0.75 * 0.5;
or
WINDOW_WIDTH = 0.75 * 0.5;

Reply:
I've used both in modern JS projects and have no preference.  Both are pretty well used.

Reply:
Ok I will go back and change.